### ISSUE TITLE
 Setting expand:false on textbox in vertical orientation gives it no height, setting true gives way too much.

### ISSUE THREAD

**Issue by leier69** [davatorium/rofi#1310]
When I set `expand: true;` It takes up way too much space and when I set expand to false, it takes up no space. why?

I've seen some other similar issues but there is never a clear fix.

config:
```
#mainbox {
    border: 0;
    padding: 0;
    children: [ bitwarden-box, inputbar, listview ];
}

#bitwarden-box {
    children: [ textbox-bitwarden ];
    expand: false;
}

#textbox-bitwarden {
    text-color: @normal-foreground;
    str: "Bitwarden";
    content: "";
    font: "Hack Bold 55";
    padding: 1;
}
```

The point is to try to make the bitwarden logo on top of my usual theme.

with expand false:
![image](https://user-images.githubusercontent.com/60528788/117727108-e146b000-b1e7-11eb-92c7-4db2b8ee838c.png)

with expand true:
![image](https://user-images.githubusercontent.com/60528788/117727177-fb808e00-b1e7-11eb-9f52-9a816ea97cfc.png)

**Comment by DaveDavenport** [comment]
> I've seen some other similar issues but there is never a clear fix.

Do you have issue number? so we can link them.

**Comment by leier69** [comment]
@DaveDavenport Thanks for getting back to me so quickly, then its most likely some stupid mistake in my config. I'll try to fill in with some more information.
```
❯ rofi -v
Version: 1.6.1
```

Entire config:
```
configuration {
    show-icons: false;
    monitor: "-1";
    lines: 20;
    fake-transparency: true;
    columns: 1;
    modi: "window,run,ssh,drun";
    location: 0;
    case-sensitive: false;
}

* {
    active-background: #58344C;
    active-foreground: @foreground;
    normal-background: @background;
    normal-foreground: @foreground;
    urgent-background: #1C304B;
    urgent-foreground: @foreground;

    alternate-active-background: @background;
    alternate-active-foreground: @foreground;
    alternate-normal-background: @background;
    alternate-normal-foreground: @foreground;
    alternate-urgent-background: @background;
    alternate-urgent-foreground: @foreground;

    selected-active-background: #BE183D;
    selected-active-foreground: @foreground;
    selected-normal-background: #007FFF;
    selected-normal-foreground: @background;
    selected-urgent-background: #839BBC;
    selected-urgent-foreground: @foreground;

    background-color: @background;
    background:     #0d1117;
    foreground:     #b0bfc4;
    border-color:   #121a27;
    spacing:        0;
    font:           "Hack 12";
}

#window {
    background-color: @background;
    width: 35%;
    border: 4;
    border-color: #30363d;
    padding: 2.5ch;
    border-radius: 15;
}

#mainbox {
    border: 0;
    padding: 0;
    children: [ bitwarden-box, inputbar, listview ];
}

#bitwarden-box {
    children: [ textbox-bitwarden ];
    expand: true;
}

#textbox-bitwarden {
    text-color: @normal-foreground;
    str: "Bitwarden";
    content: "";
    font: "Hack Bold 55";
    padding: 1;
}

#textbox {
    text-color: @foreground;
}

#inputbar {
    children:   [ entry ];
}

#listview {
    fixed-height: 0;
    border: 2px 0px 0px;
    border-color: @border-color;
    spacing: 2px;
    scrollbar: false;
    padding: 2px 0px 0px;
}

#element {
    border: 0;
    padding: 1px;
}

#element.normal.normal {
    background-color: @normal-background;
    text-color: @normal-foreground;
}

#element.normal.urgent {
    background-color: @urgent-background;
    text-color: @urgent-foreground;
}

#element.normal.active {
    background-color: @active-background;
    text-color: @active-foreground;
}

#element.selected.normal {
    background-color: @selected-normal-background;
    text-color: @selected-normal-foreground;
}

#element.selected.urgent {
    background-color: @selected-urgent-background;
    text-color: @selected-urgent-foreground;
}

#element.selected.active {
    background-color: @selected-active-background;
    text-color: @selected-active-foreground;
}

#element.alternate.normal {
    background-color: @alternate-normal-background;
    text-color: @alternate-normal-foreground;
}

#element.alternate.urgent {
    background-color: @alternate-urgent-background;
    text-color: @alternate-urgent-foreground;
}

#element.alternate.active {
    background-color: @alternate-active-background;
    text-color: @alternate-active-foreground;
}

#sidebar {
    border: 2px 0px 0px;
    border-color: @border-color;
}

#button {
    text-color: @normal-foreground;
}

#button.selected {
    background-color: @selected-normal-background;
    text-color: @selected-normal-foreground;
}

#inputbar {
    spacing: 0;
    text-color: @normal-foreground;
    padding: 1px;
}

#entry {
    spacing: 0;
    text-color: @normal-foreground;
}
```

running X, specifically bspwm. Launching rofi with either `rofi -show drun` or `rofi -dmenu`

**Comment by DaveDavenport** [comment]
Thanks for the information, this allowed me to debug this. 
With latest git it looks like:
![rofi-2021-05-11-1137-00000](https://user-images.githubusercontent.com/406245/117794149-41bc0880-b24d-11eb-933b-cfc882194223.png)

I think this is the issue has been resolved in GIT and this is a duplicate of #1247.
